.github/workflows/build.yml

```diff
@@ -27,15 +27,24 @@ jobs:
   build:
     continue-on-error: ${{ endsWith(inputs.python-version, '-dev') || contains(fromJSON('["3.7", "pypy3.7"]'), inputs.python-version) }}
     runs-on: ${{ inputs.os }}
+    if: ${{ !(startsWith(inputs.python-version, 'graalpy') && startsWith(inputs.os, 'windows')) }}
     steps:
       - uses: actions/checkout@v4
 
       - name: Set up Python ${{ inputs.python-version }}
+        if: ${{ !startsWith(inputs.python-version, 'graalpy') }}
         uses: actions/setup-python@v4
         with:
           python-version: ${{ inputs.python-version }}
           architecture: ${{ inputs.python-architecture }}
 
+      - name: Set up GraalPy ${{ inputs.python-version }}
```

Comment:
Using setup-python@main is temporary until setup-python tags a new release with GraalPy.